ghost/core/core/server/data/schema/schema.js

```diff
@@ -1059,6 +1059,18 @@ module.exports = {
         type: {type: 'string', maxlength: 50, nullable: false},
         filter: {type: 'text', maxlength: 1000000000, nullable: true},
         feature_image: {type: 'string', maxlength: 2000, nullable: true},
+        og_image: {type: 'string', maxlength: 2000, nullable: true},
+        og_title: {type: 'string', maxlength: 300, nullable: true},
+        og_description: {type: 'string', maxlength: 500, nullable: true},
+        twitter_image: {type: 'string', maxlength: 2000, nullable: true},
+        twitter_title: {type: 'string', maxlength: 300, nullable: true},
+        twitter_description: {type: 'string', maxlength: 500, nullable: true},
+        meta_title: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 300}}},
```

Comment:
It's a "soft limit"  which is set in all "meta_title" fields. More context [here](https://github.com/TryGhost/Ghost/issues/8143). My understanding it that we ha put a larger field size of 2000 chars to have a buffer just in case we ever need to increase the length of the field. In reality it's been 6 years and there was no need for increase. If we are concerned about optimal field sizes, might be a good idea to trim all `meta_title` fields to a length of 300 in Ghost v6 🤔

Reply:
🤔 I see

Yeah that seems weird we don't do that with any other fields!